lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/telemetry/LogMessage.java

```diff
@@ -0,0 +1,3 @@
+package org.enso.logging.service.logback.telemetry;
+
+public record LogMessage(String loggerName, String message, Object[] arguments) {}
```

Comment:
Can we remove the `LogMessage` and just use `ILoggingEvent`?

Reply:
It is harder to test it if we do that. I introduced `LogMessage` just because of testing. I found no way how to construct `ILoggingEvent` from tests.

Reply:
OK, in such case we could get rid of `LogFormatter` and move its only method to `LogMessage`. Not really important.